pkg/provider/provider.go

```diff
@@ -90,6 +90,7 @@ func Provider() *schema.Provider {
 			"snowflake_task":                   resources.Task(),
 			"snowflake_table":                  resources.Table(),
 			"snowflake_table_grant":            resources.TableGrant(),
+			"snowflake_stream":                 resources.Stream(),
```

Comment:
Can you put this in alphabetical order? So on line 86?

Reply:
done